WooCommerce/Classes/ViewRelated/Orders/OrdersViewController.swift

```diff
@@ -5,11 +5,28 @@ import WordPressUI
 import SafariServices
 import StoreKit
 
+protocol OrdersViewControllerDelegate: class {
+    /// Called when `OrdersViewController` is about to fetch Orders from the API.
+    ///
+    func ordersViewControllerWillSynchronizeOrders(_ viewController: OrdersViewController)
```

Comment:
You know what? I missed a very important thing. The method names start with `ordersViewController*`. 🤦‍♂ So that may be good enough and we can remove the argument. What do you think?

Reply:
TIL - I haven't used delegates for a while (mostly closures nowadays), and I have missed the Swift style guide on the delegates naming convention that you linked from raywenderlich.com. Honestly, I was a bit confused by your example of `sync()` myself when I started as well since it feels like a function from the class, not a delegate implementation.

for this case `func ordersViewControllerWillSynchronizeOrders(...)`, I think it's worth following the delegates naming convention even though the function name already implies that it's `OrdersViewController` 👍